src/ai_marketplace_monitor/webui/static/app.css

```
@@ -0,0 +1,651 @@
+  cursor: pointer;
+  border-left: 2px solid transparent;
+}
+.log-row:hover { background: var(--bg-elev-2); }
+.log-row.expanded { background: var(--bg-elev-2); border-left-color: var(--accent); }
+.log-row .log-time { color: var(--text-dim); }
+.log-row .log-level { font-weight: 600; text-transform: uppercase; font-size: 10px; padding-top: 1px; }
+.level-DEBUG .log-level { color: var(--debug); }
+.level-INFO .log-level { color: var(--info); }
+.level-WARNING .log-level { color: var(--warn); }
+.level-ERROR .log-level, .level-CRITICAL .log-level { color: var(--err); }
+.log-row .log-msg { white-space: pre-wrap; word-break: break-word; }
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)